### PR TITLE
🔁 Update: AnimeMusicQuiz - MangaWorld - AnimeWorld

### PR DESCRIPTION
**Pull request by TellenOfficial**
**Added .ts file to MangaWorld and AMQ and removed duplicate declarations for AMQ.**
**Added video to AnimeWorld and fixed a bug.**

_I don't know what happened to AnimeMusicQuiz's Presence, but it's no longer in the original repo._

_I had trouble creating the PullRequest because it turned out that I had edited 1600 files. So I deleted my old fork and made a new one._

Anime Music Quiz:
image

MangaWorld:
image

AnimeWorld:
image

Updated: 

image
image

image
image

image
image

### CODE REVIEW

**Comment by doomlerd** [comment]
Remove the mediaKeys and recompile.

**Comment by TellenOfficial** [comment]
> and don't edit tsconfig.json

What i have to do? i removed mediakeys but it say me i need to change something but idk what

File: Anime Music Quiz/presence.ts [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                timeRemaining = document.querySelector("#qpHiderText").textContent;
+                data.smallImageKey = "gamepad",
+                data.smallImageText = "Round: " + actualRoundNumber + "｜Countdown: " + timeRemaining;
+                data.details = "Round: " + totalRoundNumber,
+                data.state = "Time remaining: " + timeRemaining,
+                presence.setActivity(data);
+                }
+                }
+                }
+                }
```
Comment (Bas950) [inline Anime Music Quiz/presence.ts:79]: Looks like something is wrong here, or your formatting is really strange

File: Anime Music Quiz/presence.ts [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                timeRemaining = document.querySelector("#qpHiderText").textContent;
+                data.smallImageKey = "gamepad",
+                data.smallImageText = "Round: " + actualRoundNumber + "｜Tempo rimanente: " + timeRemaining;
+                data.details = "Round: " + totalRoundNumber,
+                data.state = "Tempo rimanente: " + timeRemaining,
+                presence.setActivity(data);
+                }
+                }
+                }
+                }
```
Comment (Bas950) [inline Anime Music Quiz/presence.ts:177]: same here

Comment (TellenOfficial) [reply Anime Music Quiz/presence.ts:177]: it's max 4 tabs...

Comment (TellenOfficial) [reply Anime Music Quiz/presence.ts:177]: i had to remove the tabs

**Comment by doomlerd** [comment]
Still not formatted correctly.

**Comment by TellenOfficial** [comment]
This is after formatted XD

**Comment by doomlerd** [comment]
Update your fork.

**Comment by TellenOfficial** [comment]
All done. Sorry for disturbe and problem @Bas950 and @doomlerd 😞. Accept my apologies.

**Comment by TellenOfficial** [comment]
image
image

image
image

image
image

@doomlerd

**Pull request merged**